src/abstracts/Command.ts

```diff
@@ -5,11 +5,13 @@ abstract class Command {
 	private readonly name: string;
 	private readonly description: string;
 	private readonly options: CommandOptions | undefined;
+	private readonly aliases: Array<string>;
 
-	protected constructor(name: string, description: string, options?: CommandOptions) {
+	protected constructor(name: string, description: string, options?: CommandOptions, aliases: Array<string> = []) {
```

Comment:
I think it might be useful to move aliases into the `CommandOptions`. Keeps the `constructor` shorter.

Reply:
Nice, I didn't notice you could do that either.